src/home-page/Content/Controls/CategoryControls.tsx

```diff
@@ -0,0 +1,13 @@
+import { ButtonBase } from '@shared';
+
+export const CategoryControls = () => {
+  return (
+    <>
```

Comment:
This doesn't seem to be related to your `MovieGenre` enum. To stay consistent, I would rather keep these connected:
```ts
return Object
   .values(MovieGenre)
   .map(genre => (
      <ButtonBase key={genre} className='category-button'>{genre}</ButtonBase>
   ))
```

Reply:
Yeah, probably. I think it will be implemented as a part of one of next HW